tests/Frontend/PathSpec.hs

```diff
@@ -33,6 +35,19 @@ import Network.Wai
 import Test.Hspec.Wai (get, post, shouldRespondWith)
 import qualified Test.Hspec.Wai.QuickCheck as Wai (property)
 
+isBrokenPath :: Main -> Bool
```

Comment:
As far as I know, there is no DB read involved in the `Paths and handlers` test suite, it uses the `mockAulaMain` uses only arbitrary instances to generate some mock data for the handler. I am opening an issue about it.